src/python/grpcio/grpc/_observability.py

```diff
@@ -174,6 +178,9 @@ def set_stats(self, enable: bool) -> None:
         """
         self._stats_enabled = enable
 
+    def save_registered_method(self, method_name: bytes):
+        self._registered_methods.add(method_name)
```

Comment:
Ideally, plugins shouldn't be accessing underscore-prefixed attributes in the base class. The interface should be method-only. We're already doing this for `tracing_enabled` (via the `@property` decorator). This is somewhat less important for backward-compatibility purposes since we're pinning the version of `grpcio-observability` to be the same as `grpcio`, but it's good practice in general.

Reply:
Ping.